src/Common/src/Interop/Windows/Winsock/ProtocolFamily.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Net.Sockets
     ///       class can use.
     ///    </para>
     /// </devdoc>
-    public enum ProtocolFamily
+    internal enum ProtocolFamily
```

Comment:
Yes, for now, as the type isn't exposed by any contract. There are teams that may require using this type internally - we can switch back to public.
Doing this per our decision to be careful about public types in Common.